CONTRIBUTING.md

```diff
@@ -18,13 +16,103 @@ Before you get started, here are some suggestions:
  3. `npm test`
  4. `make build-node`
 
-## Licencing
+## Making A Change
 
- - Every file must have a licence block at the top. This is enforced using `uber-licence`
```

Comment:
What is the replacement for uber-license?  I don't see any changes to `package.json`, how did you perform replacement of the license headers? I think we need
  * a validation step that fails the build if files don't have a license header
  * optionally, an auto-formatter that adds the headers to new files
  *

Reply:
Will investigate to see what options are available.

Reply:
perl :) - had a look at uber-license by it only uses MIT license. Will have a look for an automated checker/formatter tomorrow - hopefully we can find something suitable.

Reply:
> perl

can you post it?  Might come in handy in other repos.

Reply:
Sure: https://gist.github.com/objectiser/81491af29de2d1043764425f9c7d90a6